docs/modules/ROOT/pages/README.adoc

```diff
@@ -1,7 +1,21 @@
 image:payara-logo-blue.png[Payara Logo]
 
+== Introduction
+
+This is the documentation for the Payara Platform Enterprise Edition, designed to increase stability and security in production environments and provide customers with access to production-oriented features, tools, and capabilities.
+
+Payara Platform Enterprise which has a monthly release cadence, is designed for production environments and new features are added to Payara Platform Enterprise after they have been thoroughly tested for stability.
+
+== Download
+
+The Enterprise Edition downloads can be found in the _Payara Enterprise_ repository of our https://nexus.payara.fish[Nexus Server]. More information can be found in the https://support.payara.fish/hc/en-gb/articles/203619992-Payara-Patch-Releases-Download-Guide?utm_campaign=Payara%20patch%20Notes[Payara Enterprise Download guide] (only accessible for Enterprise Customers).
```

Comment:
The link for the Zendesk article is this one as it was renamed: https://support.payara.fish/hc/en-gb/articles/203619992-Payara-Enterprise-Download-Guide. At the moment it's in draft but it'll get published soon.